src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramServlet.scala

```diff
@@ -14,60 +16,58 @@ import org.broadinstitute.dsde.workbench.avram.util.AvramResult.unsafeRun
 import scala.concurrent.ExecutionContext
 
 
-trait AvramServlet {
+abstract class AvramServlet(avram: Avram) {
 
   implicit val executionContext: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
 
   val log: Logger = Logger.getLogger(getClass.getName)
-  private val samDao = Avram.samDao
+  private val samDao = avram.samDao
   private val bearerPattern = """(?i)bearer (.*)""".r
 
-  def handleAuthenticatedRequest[T](request: HttpServletRequest, response: HttpServletResponse)
+  def handleAuthenticatedRequest[T](bearerToken: String)
```

Comment:
Maybe name this `authorizationHeader` -- we need to call `extractBearerToken` to actually get the bearer token.